### PR TITLE
Add entity classes

### PR DESCRIPTION
**Pull request by jpmartin2**
Added all of the entity classes from the UML. Not all methods are currently implemented, but are at least stubbed out.

### CODE REVIEW

**Comment by jkuszmaul** [comment]
No documentation.

File: src/kabasuji/models/Bullpen.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package kabasuji.models;
+
+import java.util.ArrayList;
+
+public class Bullpen {
+  ArrayList<Piece> pieces;
+
```
Comment (PeterMitrano) [inline src/kabasuji/models/Bullpen.java:7]: we need a no argument constructor for level builder

File: src/kabasuji/models/GameModel.java [hunk 2]
```diff
@@ -1,19 +1,47 @@
...
 
 public class GameModel {
 
   public int numLevels = 15;
   public Level[] levels;
+  
+  Level activeLevel;
+  ArrayList<Achievement> achievements;
+  //ArrayList<IMove> undoStack;
+  //ArrayList<IMove> redoStack;
```
Comment (PeterMitrano) [inline src/kabasuji/models/GameModel.java:15]: we should uncomment these eventually

File: src/kabasuji/models/GameModel.java [hunk 3]
```diff
@@ -1,19 +1,47 @@
 package kabasuji.models;
 
+import java.util.ArrayList;
+
 import kabasuji.models.Level;;
 
 public class GameModel {
 
   public int numLevels = 15;
   public Level[] levels;
```
Comment (PeterMitrano) [inline src/kabasuji/models/GameModel.java]: needs to be array list

Comment (PeterMitrano) [inline src/kabasuji/models/GameModel.java:10]: needs to be array list

File: src/kabasuji/models/LevelIndex.java [hunk 4]
```diff
@@ -0,0 +1,9 @@
+package kabasuji.models;
+
+public class LevelIndex {
```
Comment (PeterMitrano) [inline src/kabasuji/models/LevelIndex.java:3]: this class is dumb

**Pull request merged**